code/src/components/Question1.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Question1 = ( {name, onNameChange}) => {
+
+    return (
+    <>
+    <p>Start by entering your name  
+        <span role="img" aria-label="Smile Face Emoji">👇</span>  
+    </p>
+            <div>
+                <label htmlFor="name">Name: </label>
+                <input
+                    type='text'
+                    onChange={(e) => onNameChange(e)}
+                    required
```

Comment:
The basic HTML validation is working fine for both the text input and the select!